### DISCUSSION TITLE
Show the tabs as a tree structure in the side bar and organise tabs in groups and folders

### DISCUSSION THREAD

**Discussion by sirinath** [zen-browser/desktop#1078] [Ideas]
Please show the tabs as a tree structure in the side bar, where you can see the parent child relationships. When I open a link as a new tab from a existing tab this relationship is created.

If all tabs a top level they can be as they are, but a thin coloured line can be used to group parent child relationships.

Also ability to group and organise tabs into groups or folders in the side bar. We can give a group / tab its own icon and title.

**Comment by GitMurf** [comment]
FYI see here for a Discussion one of the maintainers opened up to follow this development: https://github.com/zen-browser/desktop/discussions/891

**Comment by universish** [comment]
Is the feature you are talking about the one in brave browser on android? Opening tabs in a group. Do I understand correctly?

**Reply by vardecab** [reply]
> Is the feature you are talking about the one in brave browser on android? Opening tabs in a group. Do I understand correctly?

Something like this (screenshot from Orion browser): 
<img width="195" alt="SCR-20241113-cfau" src="https://github.com/user-attachments/assets/8ab3c2c7-e90f-428d-8065-e2661e0f321f">
